### PR TITLE
The two 68040RC (normal & V)

### PR DESCRIPTION
**Pull request by rdolbeau**
The 'normal' version exists commercially, but apparently the 'V' version (static core, 3.3V supply, no FPU) doesn't.

### CODE REVIEW

File: pga_socket.scad [hunk 1]
```diff
@@ -4,7 +4,7 @@
 /*[Variant]*/
 
 // Which socket you need
-pga_which = "128"; // ["12","64","68-10x10":68 10x10 - MC68000,"68-11x11":68 11x11 - 80C286,"84","114","128":128 - MC68030,"132","176","223","238"]
+pga_which = "128"; // ["12","64","68-10x10":68 10x10 - MC68000,"68-11x11":68 11x11 - 80C286,"84","114","128":128 - MC68030,"132","179" - MC68040RCxx, "182" - MC68040RCxxV,"223","238"]
```
Comment (mmuman) [inline pga_socket.scad]: Syntax is wrong there, you need to have `"value":description`, so `"179":179 - MC68040RCxx` and so on.

**Review by mmuman — COMMENTED** [review]
Thanks, can you fix the syntax in the comment? It's actually used for the customizer to generate the drop-down so it must be conform.

File: pga_socket.scad (hunk 1 above)
Comment (rdolbeau) [reply pga_socket.scad]: Didn't realize it was more than a comment, should be fixed

**Comment by mmuman** [comment]
Damn, a squash would have been enough…

**Comment by mmuman** [comment]
Thanks!

**Comment by mmuman** [comment]
Oh the 176 one got dropped in the comment… I'll add it back.

**Pull request merged**